backend/app/assets/javascripts/spree/backend/variant_autocomplete.js.coffee.erb

```diff
@@ -1,10 +1,8 @@
 # variant autocompletion
 $(document).ready ->
-  if $("#variant_autocomplete_template").length > 0
-    window.variantTemplate = Handlebars.compile($("#variant_autocomplete_template").text())
-    window.variantStockTemplate = Handlebars.compile($("#variant_autocomplete_stock_template").text())
-    window.variantLineItemTemplate = Handlebars.compile($("#variant_line_items_autocomplete_stock_template").text())
-  return
+  window.variantTemplate = HandlebarsTemplates["variants/autocomplete"]
```

Comment:
@richardnuno Aye, I believe the goal is to have all templates compiled and available on each page.